src/main/java/seedu/address/model/person/patient/Remark.java

```diff
@@ -1,15 +1,20 @@
 package seedu.address.model.person.patient;
 
 import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
 
 /**
  * Represents any remarks for a person in the address book.
  * Guarantees: immutable; is always valid
  */
 public class Remark {
 
-    public static final String MESSAGE_CONSTRAINTS = "Add any remarks for the patient.";
-
+    public static final String MESSAGE_CONSTRAINTS =
```

Comment:
Is there a reason remark should not be blank?

Reply:
Agreed, remark should be optional

Reply:
Remark should be optional but should not be blank if it is included as a field